contrib/build/win/_build.sh

```diff
@@ -4,26 +4,49 @@
 
 set -e  # Exit on error
 
-if [ -z "$1" ] || [ -z "$2" ]; then
-    echo "Please pass Fulcrum and rocksdb dirnames as the two args"
+if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
+    echo "Please pass Fulcrum, rocksdb, and jemalloc dirnames as the three args"
     exit 1
 fi
 
 PACKAGE="$1"
 ROCKSDB_PACKAGE="$2"
+JEMALLOC_PACKAGE="$3"
 TARGET_BINARY=Fulcrum.exe
 TARGET_ADMIN_SCRIPT=FulcrumAdmin
 
 top=/work
 cd "$top" || fail "Could not cd $top"
 . "$top/$PACKAGE/contrib/build/common/common.sh" || (echo "Cannot source common.h" && exit 1)
 
+info "Running configure for jemalloc ..."
+cd "$JEMALLOC_PACKAGE" || fail "Could not change dir to $JEMALLOC_PACKAGE"
+CXX=x86_64-w64-mingw32.static-g++ LD=x86_64-w64-mingw32.static-ld CC=x86_64-w64-mingw32.static-gcc-7.5.0 \
+    ./autogen.sh --host x86_64-w64-mingw32 -with-jemalloc-prefix= --disable-shared --enable-static \
```

Comment:
Same with the single dash here.

Reply:
Oh lol. Should be double-dash right?

Reply:
Yeah ok fixed.  Weird that it worked.. hmm.